gradle/manifest-gen.gradle

```diff
@@ -12,18 +12,10 @@ else if (baseVersion.split('\\.').length == 4)
 	baseVersion = baseVersion.substring(0, baseVersion.lastIndexOf('.'))
 def qualifiedVersion = baseVersion + '.v' + buildTime
 
-jar.manifest {
-	version qualifiedVersion
-	instruction 'Import-Package', 'org.apache.log4j.*;version="1.2.15"'
-	instruction 'Import-Package', 'com.google.common.*;version="[14.0,22.0)"'
-	instruction 'Import-Package', 'org.eclipse.core.runtime.*;version=!;common=!'
-	instruction 'Import-Package', "org.eclipse.xtext.xbase.lib.*;version=\"${version}\""
-	instruction 'Import-Package', '!org.eclipse.xtext.web.example.*'
-	instruction 'Import-Package', '!org.eclipse.xtend2.lib.*'
-	instruction 'Import-Package', '!org.eclipse.xtext.junit4.*'
-	instruction 'Import-Package', '!org.hamcrest.*'
-	instruction 'Import-Package', '!org.junit.*'
-}
+jar.bnd (
+	'Bundle-Version': qualifiedVersion,
+	'Import-Package': "org.apache.log4j.*;version='1.2.15',com.google.common.*;version='[14.0,22.0)',org.eclipse.core.runtime.*;version=!;common=!,org.eclipse.xtext.xbase.lib.*;version=\"${version}\",!org.eclipse.xtext.web.example.*,!org.eclipse.xtext.idea.example.*,!org.eclipse.xtend2.lib.*,!org.eclipse.xtext.junit4.*,!org.hamcrest.*,!org.junit.*"
```

Comment:
am not sure if our log4j versions are consitent. what do we use in the bom?

Reply:
1.2.17

Reply:
Done